content/account_management/faq.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Account management FAQ
+kind: documentation
+autotocdepth: 2
+customnav: accountmanagementnav
+---
+
+### How do I setup my team in Datadog?
+
+The admin of the account should enter the email addresses of team members
+[here][other-1]. Some team best practices are as follows:
+
+* When the team member receives the confirmation email, they will be provided with a link to log in directly. The user should not click ‘sign up’ during this process.
+* If multiple users from the same organization sign up separately, this will register as different organizations in Datadog. Please reach out to support to have these merged, but please note that all information contained in the account getting merged will not be transferred over.
```

Comment:
If multiple users from the same organization sign up separately, it will create multiple organizations which do not share data. If your team has created multiple organizations and would like to merge them, support can assist in doing so.  However, please note that only users are transferred. Data such as metrics or monitors created in the account being merged will not be transferred.

Reply:
👍